apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -110,18 +110,20 @@ NetSimLocalClientNode.inherits(NetSimClientNode);
 /**
  * Static async creation method. See NetSimEntity.create().
  * @param {!NetSimShard} shard
+ * @param {string} displayName
  * @param {!NodeStyleCallback} onComplete - Method that will be given the
  *        created entity, or null if entity creation failed.
  */
-NetSimLocalClientNode.create = function (shard, onComplete) {
-  NetSimEntity.create(NetSimLocalClientNode, shard, function (err, node) {
+NetSimLocalClientNode.create = function (shard, displayName, onComplete) {
+  var templateNode = new NetSimLocalClientNode(shard);
+  templateNode.displayName_ = displayName;
+  templateNode.getTable_().create(templateNode.buildRow_(), function (err, row) {
```

Comment:
I got to spend a bunch of time helping document. the word binary document format, where they have things like this https://msdn.microsoft.com/en-us/library/dd944596(v=office.12).aspx (almost certainly not the worst offender)